### ISSUE TITLE
[Feature Request] Generating multiple keys/messages at once

### ISSUE THREAD

**Issue by Zaimwa9** [opral/inlang-sherlock#132]
Following my bug issue forcing me to create empty messages.

It would be nice to be able to generate XX messages at once instead of spamming Create Message when we need several messages

**Comment by Zaimwa9** [comment]
I also think it would help new users migrate from their existing translation tool

**Comment by felixhaeberle** [comment]
I'll look into this!:)

**Comment by Zaimwa9** [comment]
Hi Felix, thanks for your reply. Basically, I'll split into 2 use cases.

The first one was that we migrated from an existing tool, meaning we needed to generate thousands of keys (maybe we tackled it wrong). 
It would have been just great to be able to generate at least 10 keys at once when migrating a file and then just fill them.

Second use case (nice to have), daily usage
1. Cmd+shit+P Sherlock > create multiple messages > XX
2. I am working on a new feature, knowing i'll have a dozen of translated texts, I want to have the keys available before starting to put the content
3. Could be included in Sherlock, could be in fink
4. Cf 1

Thanks for your answer. It's mostly a feedback following the migration + the bug I had with sherlock in the other issue that made it very painful to generate keys

**Comment by felixhaeberle** [comment]
@Zaimwa9 Thanks for answering that fast and for the clarification. This is great feedback and let us plan ahead to support multiple/bulk message creation. The next step would be to plan the implementation, I would most likely guess it should belong somewhere in https://github.com/opral/monorepo/blob/main/inlang/packages/sherlock/src/commands/createMessage.ts – or a complete new command! 

As we have other things on our list, too, I can't give an estimate for when this feature will be shipped, but I would be happy to accept an PR.
